ui/apps/platform/src/Containers/NetworkGraph/components/StyleNode.tsx

```diff
@@ -90,13 +110,26 @@ const renderDecorator = (
     const { x, y } = getShapeDecoratorCenter
         ? getShapeDecoratorCenter(quadrant, element)
         : getDefaultShapeDecoratorCenter(quadrant, element);
+    const padding =
+        quadrant === TopologyQuadrant.lowerLeft
+            ? DEFAULT_DECORATOR_PADDING - 2.5
+            : DEFAULT_DECORATOR_PADDING;
```

Comment:
What is the magic number `2.5` ?

Reply:
unfortunately the `getShapeDecoratorCenter` does not actually center our custom SVGs and i'm not sure how it calculates the center (i think the size of our SVGs have something to do with it), but adding the padding seemed to make sure that the SVG looks more centered so I added it in for now. I can add it as a more descriptive variable so it's more clear